src/test/java/org/dependencytrack/tasks/CloneProjectTaskTest.java

```diff
@@ -67,7 +67,7 @@ public void testCloneProjectDoesNotExist() {
                     assertThat(state.getStatus()).isEqualTo(FAILED);
                     assertThat(state.getStartedAt()).isNotNull();
                     assertThat(state.getUpdatedAt()).isBefore(Date.from(Instant.now()));
-                    assertThat(state.getFailureReason()).contains("Project with UUID " + uuid + " was supposed to be cloned, but it does not exist anymore");
```

Comment:
UUID is no longer included here, because it's already part of the logging context (MDC), and for the workflow it's redundant information anyway.